File: android/snlogger/src/main/java/com/vd/snlogger/util/SNLogUtils.java
package com.vd.snlogger.util;


import com.vd.snlogger.SNLogger;
import com.vd.snlogger.base.T;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;


public class SNLogUtils {
    public static String parseContent(@NonNull String content, @Nullable SNLogMethodInfo methodInfo) {
        if (methodInfo != null) {
            return "[" +
                    Thread.currentThread().getId() +
                    "](" +
                    methodInfo.clsName +
                    ":" +
                    methodInfo.lineNumber +
                    ")" +
                    methodInfo.methodName +
                    ": " +
                    content;
        }
        return content;
    }

    public static String parseTAG(T appTag) {
        return SNLogger.TAG + "::" + appTag.toString();
    }
}

File: android/snlogger/src/main/java/com/vd/snlogger/util/SNLogUtils.kt
package com.vd.snlogger.util

import com.vd.snlogger.SNLogger
import com.vd.snlogger.base.T

class SNLogUtils {
    companion object {
        fun parseContent(content: String, methodInfo: SNLogMethodInfo?): String {
            if (methodInfo != null) {
                return "[" + Thread.currentThread().id + "](" + methodInfo.clsName + ":" + methodInfo.lineNumber + ")" + methodInfo.methodName + ": " + content
            }
            return "[" + Thread.currentThread().id + "]" + ": " + content;
        }

        fun parseTAG(appTag: T): String {
            return SNLogger.TAG + "::" + appTag.toString()
        }
    }
}